### DISCUSSION TITLE
How to retrieve the conversation comments of a PR?

### DISCUSSION THREAD

**Discussion by mickael-h** [actions/github-script#416] [Q&A]
The documentation only gives a way to retrieve the **review** comments:
https://octokit.github.io/rest.js/v20#pulls-list-review-comments
Or to get comments on **issues**:
https://octokit.github.io/rest.js/v20#issues-list-comments

There doesn't seem to be a way to retrieve the conversations on a pull request.

**Comment by joshmgross** [comment]
@mickael-h could you clarify what you're looking for here?

The comments on issues API should work for pull requests - https://docs.github.com/en/rest/issues/comments?apiVersion=2022-11-28#about-issue-and-pull-request-comments

**Reply by mickael-h** [reply]
>  The comments on issues API should work for pull requests

Talk about confusing... My first attempt doing this didn't work, so I assumed it was because pull requests are not issues.
But since you're telling me it should work anyway, I'll give it another shot.
